packages/synapse-react-client/src/components/styled/StyledTableContainer.tsx

```diff
@@ -1,8 +1,12 @@
 import { Box, BoxProps, styled } from '@mui/material'
 import { StyledComponent } from '@emotion/styled'
 
-export const StyledTableContainer: StyledComponent<BoxProps> = styled(Box)(
-  ({ theme }) => ({
+export type StyledTableContainerProps = BoxProps & {
+  density?: 'default' | 'compact'
+}
```

Comment:
Added `density` prop to make this more easily reusable and remove some common style overrides